lnrpc/rpc.proto

```diff
@@ -2081,11 +2081,26 @@ message WalletBalanceResponse {
 message ChannelBalanceRequest {
 }
 message ChannelBalanceResponse {
-    // Sum of channels balances denominated in satoshis
-    int64 balance = 1;
+    // Deprecated. Sum of channels balances denominated in satoshis
+    int64 balance = 1 [deprecated = true];
 
-    // Sum of channels pending balances denominated in satoshis
-    int64 pending_open_balance = 2;
+    // Deprecated. Sum of channels pending balances denominated in satoshis
+    int64 pending_open_balance = 2 [deprecated = true];
+
+    // Sum of channels local balances denominated in satoshis.
+    int64 local_balance = 3;
```

Comment:
would you mind adding a `_sat` suffix to these new fields? typically we try to specify the units for amounts (either `_msat` vs `_sat`)

Reply:
perhaps there is also a larger question of whether we want to expose this as sat or msat (also goes for unsettled balance) for channel balances. currently the unsettled_balance will show zero for sub-sat payments. what do you think @bhandras?

Reply:
Problem with not exposing `msat` is that people doing accounting using the `sat` fields might end up with results that don't add up.

Reply:
@yyforyongyu discussed offline, but i think the move is to expose both `_sat` and `_msat`

Reply:
So we want to expose both `local_balance_sat` and `local_balance_msat` in the response?
If we display `local_balance_sat: 10000.123` instead of a uint field, would that be helpful?

Reply:
> So we want to expose both local_balance_sat and local_balance_msat in the response?

correct

> If we display local_balance_sat: 10000.123 instead of a uint field, would that be helpful?

This would deviate from how sat/msat is displayed across the rest of the rpc interface. at the end of the day these are integral values, and we should encourage people to treat them as such. if they want to display them differently that is ultimately up to them

Reply:
I tried to add both `_msat` and `_sat` to all the fields. However, I found it a bit repetitive so I made a new field `Amount` to standardize how the balances are treated. Let me know what you think. I sense the next move is to uniform the sat/msat info in all balance fields?
Probably unrelated to this PR, the current balance fields are using `int64`, I'm wondering why. So the values may be negative?

Reply:
@yyforyongyu i like this approach a lot, i think it is much cleaner than having multiple fields for each denomination. my only concern is that a bit of a departure from how we do this elsewhere in our rpc interfaces. that said, i think this is an appropriate place to test out this change, given that the primary purpose of this endpoint is to provide rich balance information.

>  I sense the next move is to uniform the sat/msat info in all balance fields?

I think I would hold off on that for now, perhaps we can solicit some feedback on this rpc before doing so universally? it would be a rather large change to commit to upfront, but imo this is a good starting point for that discussion!

> Probably unrelated to this PR, the current balance fields are using int64, I'm wondering why. So the values may be negative?

No the balances can't/shouldn't ever be negative, my preference is to use `uint64` for these